protocols/Imperial_College/README.md

```diff
@@ -0,0 +1,45 @@
+# Drug Mixing and Transferring
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+### Partner
+Imperial College
+
+## Categories
+* Basic Pipetting
+	* Plate Consolidation
+
+
+## Description
+This protocol allows you to add different combination of compounds from several 96-well plate to each well of a 24-well plate for mixing using a single-channel pipette. Each of these 24 wells will then be transferred to a clean 24-well plate. The compound source information and target locations are defined in a CSV file.
+
+### Time Estimate
+
+### Robot
+* [OT 2](https://opentrons.com/ot-2)
+* [OT PRO](https://opentrons.com/robots/ot-one-s-pro)
+* [OT Standard](https://opentrons.com/robots/ot-one-s-standard)  
+* [OT Hood](https://opentrons.com/robots/ot-one-s-hood)
```

Comment:
will not work with the hood because you are using 'E3' slot. Either change the trash to another location or remove hood in readme